### PR TITLE
feat(core): adjust share menu ui

### PR DESCRIPTION
**Pull request by JimmFly**
https://github.com/user-attachments/assets/6538c046-3872-4c98-a389-81b86a2978a4

### CODE REVIEW

**Comment by JimmFly** [comment]
* **#7931** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7931?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @JimmFly and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by pengx17 — APPROVED** [review]

File: packages/frontend/core/src/components/affine/share-page-modal/share-menu/share-page.tsx [hunk 1]
```diff
@@ -1,5 +1,4 @@
 import { notify, Skeleton } from '@affine/component';
-import { PublicLinkDisableModal } from '@affine/component/disable-public-link';
```
Comment (EYHN) [inline packages/frontend/core/src/components/affine/share-page-modal/share-menu/share-page.tsx:2]: delete this file?

Comment (JimmFly) [reply packages/frontend/core/src/components/affine/share-page-modal/share-menu/share-page.tsx:2]: This component is also used in doc options in All doc.

Comment (JimmFly) [reply packages/frontend/core/src/components/affine/share-page-modal/share-menu/share-page.tsx:2]: Deleted

**Pull request merged**